### DISCUSSION TITLE
fallback:true as check for custom 404

### DISCUSSION THREAD

**Discussion by ashx404** [vercel/next.js#13080] [Help]
Can fallback:true be used as a check to determine whether the router exists or not. So if isFallback is true. I can conditionally render Page doesnt exists or something else. Is this correct according to the good practices?
I tried it, it works but the code for content page also runs as 
if(router.isFallback){
return <div>404</div>
}
else{
<div>CONTENT</div>
}
I am not passing any staticProps so it is giving an error.

**Comment by bscaspar** [comment]
Hey @ashx404, the page will automatically return a 404 for routes that don't exist if you use `fallback: false`. You can check out the documentation [here](https://nextjs.org/docs/basic-features/data-fetching#the-fallback-key-required).

If the route doesn't exist it will render the Next 404 page. To customize this page you can create your own at `pages/404.js`
